docs/examples/import-https-url-definition/rabbitmq.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: import-https-url-definitions 
+spec:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 2
+      memory: 8Gi
+    limits:
+      cpu: 2
+      memory: 8Gi
+  rabbitmq:
+    additionalConfig: |
+      definitions.import_backend = https
+      definitions.https.url = https://raw.githubusercontent.com/rabbitmq/sample-configs/main/lot-of-queues/5k-queues.json
+      definitions.tls.versions.1 = tlsv1.2
+      vm_memory_high_watermark_paging_ratio = 0.85
```

Comment:
Similar question to above, is configuring memory high water mark necessary here?